src/api/solr-query.js

```diff
@@ -43,7 +43,7 @@ const fieldToQueryFilter = (field) => {
     return textFieldToQueryFilter(field);
   } else if (field.type === "list-facet") {
     return listFacetFieldToQueryFilter(field);
-  } else if (field.type === "range-facet" || field.type === "range") {
+  } else if (field.type === "range-facet" || field.type === "range" || field.type === "date-range-facet") {
```

Comment:
This piece just tells the solr querying code to treat the new `date-range-facet` type like the other range types.  With the changes to have the solr index use the date range field instead of the date point field, the `[YEAR to YEAR]` syntax works for filter queries.